### ISSUE TITLE
Provide Ministry Ability to see how Student Account was created

### ISSUE THREAD

**Issue by AnnaPBashkatova** [bcgov/SIMS#3244]
**Story Description**
As a Ministry user, I want to be able to see if Student account was created through BC Services Card or Basic BCeID so that. I can  support students in handling profile information 

**Acceptance Criteria:**

- [ ] Enable Ministry to see details on how Student account was created. 

- [ ] Enable Ministry to see information on Student account conveying that Basic BceID was switched into BC Services Card Log in method.

**Business Context:**

As a Ministry User I can not distinct between BasicBCeID and BC Services Student account once they are created. BC Services student account profile updates must be directed to Service BC and BasicBCeID dealt within the StudentAidBC. As a Ministry User I need to be able to tell which case I am dealing with.

**Comment by michesmith** [comment]
Closing-duplicate per discussion at 2x Milestone Planning

**Comment by Joshua-Lakusta** [comment]
@AnnaPBashkatova is there any chance we can reach out to Service delivery to see if this is needed. If so it would be appreciated if they can complete the template to better understand the need for this and criticality assessment.

**Comment by AnnaPBashkatova** [comment]
@Joshua-Lakusta 
Hey Josh, we just had this exact question from our finance team and with all the most recent updates to SIMS we determined that now there is an easy way to distinguish between accounts for all the user including Tier 1. The Basic BCeID accounts now have an "Edit" button beside them. The main concern that SD had and that is why I logged the ticket, is the situation when SD is on the phone with student they did not have an ability to determine if student needs to approach Service BC or Tier 2 for name and email change. This is no longer a valid concern with most recent updates and I believe we can address it through training.
